src/Test/Perf/util/runner_util.csx

```diff
@@ -38,3 +40,134 @@ async Task<ScriptState<object>> RunFile(string fileName)
     return await state.ContinueWithAsync<object>(text, scriptOptions);
 }
 
+/// Gets all csx file recursively in a given directory
+IEnumerable<string> GetAllCsxRecursive(string directoryName)
```

Comment:
Note:  If the intent was the same, I'm fine with rewriting the code above as an iterator (just make this take into account the "skip" argument).  I think the iterator implementation is a little easier to read.